test/testdata/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:latest
+FROM quay.io/app-sre/centos:latest
```

Comment:
Wondering if it would make sense to use https://registry.centos.org/centos/centos ? I don't know which one will be maintained better?

Reply:
I don't really mind. I can swap it if it's preferred, since it does look more official :)

Reply:
We rely on quay.io already from the cluster, so this seems to not introduce another unknown into the tests.

Reply:
My point was not about `quay.io`, but rather about how well `app-sre/centos:latest` is maintained. If it's updated once in a blue moon, it's not a great image to depend on.

Reply:
Updated this and other two uses of centos images with the centos' own registry. Checked if it works in case of the tests, but no the other two cases.